pages/SchedulePage.js

```diff
@@ -131,6 +134,10 @@ const styles = StyleSheet.create({
         color: '#fefefe',
         opacity: 0.7
     },
+    eventChevron: {
+        flex: 1,
+        alignItems: 'center'
```

Comment:
justifyContent center to vertically center the chevron with the row

Reply:
oh true, it looked fine on my phone but if i look closer youre right